packages/api/amplify_api_dart/lib/src/graphql/model_helpers/model_mutations.dart

```diff
@@ -114,3 +114,5 @@ class ModelMutations {
     );
   }
 }
+
+// ignore_for_file: flutter_style_todos
```

Comment:
Yes, but also this shouldn't be permanent since it's clearly just a bug in the todos check

Reply:
added TODO comments